src/styles/page-container.scss

```diff
@@ -1,5 +1,5 @@
 .page-container {
   display: grid;
   grid-template-columns: minmax(250px, max-content) auto;
-  height: 100vh;
+  min-height: 100vh;
```

Comment:
Having the height set to 100vh made it so that the page-container background only extended to the bottom of the screen view. If a user scrolled down past 100vh, the background color would default back to white.

Reply:
Makes sense, thanks

Reply:
I'll actually revert this part, it creates a regression and the sidebar can't be scrolled independently from the dashboard. Dark mode still works when scrolling.